app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for human in people:
+        person = Person(name=human["name"], age=human["age"])
+
+        # set attributes "husband" and "wife" as string-value
+        if human.get("wife"):
+            person.wife = human["wife"]
+        if human.get("husband"):
+            person.husband = human.get("husband")
+        persons.append(person)
+
+    for person in persons:
+        # set link on wife(husband) object
+        if hasattr(person, "wife"):
+            person.wife = person.people.get(person.wife)
+        elif hasattr(person, "husband"):
+            person.husband = person.people.get(person.husband)
```

Comment:
You can simplify it for a one `for` loop instead of two. 
Minor tip (as I see Dmytro also left same comment as me) - think about what is going on when you are creating an instance of `Person` and what `people` variable do 🤔.
Try, if any questions, at QnA session we are open for discussion!

Reply:
Also, `for` loop and list comprehension not the same concept from my point of view